ansible_base/lib/abstract_models/common.py

```diff
@@ -93,10 +108,23 @@ def save(self, *args, **kwargs):
                 self.created_by = user
                 update_fields.append('created_by')
 
-        if 'modified_by' not in update_fields:
-            self.modified_by = user
-            update_fields.append('modified_by')
+        return super().save(*args, **kwargs)
+
 
+class AbstractCommonModel(models.Model):
```

Comment:
Should we follow through the pattern and put the save into `EncryptedModel`? And then just have the `get_summary_fields` in a `SummaryModel` or something like that?

Reply:
We could - my thought with that was that we want the encrypted/summary/related stuff to "just work" in nearly every case, so that's why I left it here.

Basically, `AbstractCommonModel` exists because my thought process was that we still do have "common" logic that we want in nearly every model, so the idea was to just split out the stuff that that isn't true for.

I'm not sure how much it's worth taking the abstraction to the extreme, and also, we can pretty easily do that later on (since `CommonModel` will inherit from those new classes - nothing in the end-user code has to change to pick that up).

Reply:
Well, we'd lose created/modified. We'd need an intermediary like we did with CommonModel, I guess.

Reply:
I think I'm inclined to leave it as is. If we want to split this stuff up more in the future, it's easy enough to do without breaking existing contracts.